src/aleph_client/commands/instance/__init__.py

```diff
@@ -602,7 +602,8 @@ async def confidential_init_session(
         echo("Could not get the certificate from the CRN.")
         return
 
-    platform_cert_path = Path(platform_file).rename(session_dir / "platform_certificate.pem")
+    # pathlib.Path.rename raises "Invalid cross-device link" if the destination is not on the current filesystem.
+    platform_cert_path = shutil.move(platform_file, session_dir / "platform_certificate.pem")
```

Comment:
did you check the return type of shutil.move ?

Reply:
No, but the code works after the change, and the official doc mentions:
>  Changed in version 3.3: Added explicit symlink handling for foreign filesystems, thus adapting it to the behavior of GNU’s mv. **Now returns dst**.